machines/backupAndRestore/backupAndRestoreSetup.ts

```diff
@@ -158,22 +160,27 @@ export const backupAndRestoreSetupMachine = model.createMachine(
             {
               cond: 'isSignedIn',
               actions: ['setProfileInfo', 'unsetIsLoading'],
-              target: 'backupAndRestore',
+              target: 'backupAndRestore', //backupAndRestore abhishek
             },
             {
               cond: 'isNetworkError',
               actions: 'sendBackupAndRestoreSetupErrorEvent',
               target: '.noInternet',
             },
             {
-              cond: 'isIOSAndNotSignedIn',
-              actions: [
-                'unsetIsLoading',
-                'setErrorReasonAsAccountRequired',
-                'sendBackupAndRestoreSetupErrorEvent',
-              ],
+              cond: 'isAuthorised',
```

Comment:
Are we moving to error if the person is authorized?
If not should we rename the guard to 'isNotAuthorized' ? This way the guard clearly defines what is the intention

Reply:
Renamed to isAuthorisedAndCloudAccessNotGiven